Common/include/mpi_structure.inl

```diff
@@ -597,6 +609,12 @@ inline void CBaseMPIWrapper::Alltoall(void *sendbuf, int sendcount, Datatype sen
   CopyData(sendbuf, recvbuf, recvcount, sendtype);
 }
 
+inline void CBaseMPIWrapper::Alltoallv(void *sendbuf, int *sendcounts, int *sdispls, Datatype sendtype,
```

Comment:
@talbring : does everything here look ok for including Alltoallv() in the wrapper?